### ISSUE TITLE
Add an option to just swipe without crossing the threshold

### ISSUE THREAD

**Issue by TheSeamau5** [TheSeamau5/swipe-pages#4]
It would be nice if you could just transition to the next page with a fast swipe by doing something like

```
if (Math.abs(oldEvent.dx - newEvent.dx) > someNumber){
    goToNextPage();
}
```

**Comment by ohanhi** [comment]
@TheSeamau5 I could take a look at velocity-based switching. Do you have some specific breaking changes in mind for v0.3?

**Comment by TheSeamau5** [comment]
Yeah, but don't worry. I'll just do the necessary merging. The big one to consider is the support for right-to-left. Basically, you would add an attribute called `rtl : Bool` which would mean that pages appear in sequence from right to left as opposed to the current left to right. So, to swipe to the next page would mean swiping from left to right. This is just an if-statement, so, as I said, don't worry too much.
